### PR TITLE
fix: `)` position with `createParenthesizedExpressions`

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Closes #10390 
| Patch: Bug Fix?          | √
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | √
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/15515"></a>

### CODE REVIEW

File: packages/babel-generator/test/fixtures/sourcemaps/comment-before-parentheses-return-arg-createParenthesizedExpressions/output.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function assertElement(assertFn, shouldBeElement, opt_message) {
+  return (/** @type {!Ele	ment} */(
+    assertType_(
+    assertFn,
```
Comment (nicolo-ribaudo) [inline packages/babel-generator/test/fixtures/sourcemaps/comment-before-parentheses-return-arg-createParenthesizedExpressions/output.js:4]: Probably unrelated, but the indentation here is broken.

**Review by nicolo-ribaudo — APPROVED** [review]

File: packages/babel-generator/test/fixtures/sourcemaps/comment-before-parentheses-return-arg-createParenthesizedExpressions/output.js (hunk 1 above)
Comment (liuxingbaoyu) [reply packages/babel-generator/test/fixtures/sourcemaps/comment-before-parentheses-return-arg-createParenthesizedExpressions/output.js:4]: Yes, I plan to take a look at this in a follow-up PR. :)

**Comment by liuxingbaoyu** [comment]
To be honest, I'm not sure if this PR should be deferred to BABEL8 since it affects older versions of jest.
Also the issue this PR fixes is not that urgent.

**Comment by nicolo-ribaudo** [comment]
I think it is fine to land it, since it doesn't make tests with older Jest versions fail. If I understood correctly, what happens with older Jest version is that when Jest has to (re)generate some inline snapshots in a test file it might move around some parentheses in that file, right?

**Comment by liuxingbaoyu** [comment]
You're right, it shouldn't break tests that haven't changed, it just might cause extra blank lines when regenerating.

**Review by JLHwung — APPROVED** [review]

**Pull request merged**